api/turing/models/router_version.go

```diff
@@ -15,6 +16,13 @@ const (
 	RouterVersionStatusUndeployed RouterVersionStatus = "undeployed"
 )
 
+type RouterProtocol config.Protocol
```

Comment:
ah good question. I was thinking to do that or not because `config.Protocol` comes from the router's code. 

I was thinking to wrap it here so the api server code uses the type in models package instead of depending directly on the router. 

Probably overthinking, i will use `config.Protocol` directly